interfaces/acados_c/ocp_nlp_interface.c

```diff
@@ -228,6 +233,45 @@ ocp_nlp_config *ocp_nlp_config_create(ocp_nlp_plan_t plan)
             exit(1);
     }
 
+    // globalization
+    switch (plan.globalization)
+    {
+        case FIXED_STEP:
+            ocp_nlp_globalization_fixed_step_config_initialize_default(config->globalization);
+            break;
+        case MERIT_BACKTRACKING:
+            ocp_nlp_globalization_merit_backtracking_config_initialize_default(config->globalization);
+            break;
+        case FUNNEL_L1PEN_LINESEARCH:
+            ocp_nlp_globalization_funnel_config_initialize_default(config->globalization);
+            break;
+        default:
+            printf("\nerror: ocp_nlp_config_create: unsupported plan->globalization\n");
+            exit(1);
+    }
+
+    // globalization step update
+    // NLP solver
+    switch (plan.nlp_solver)
+    {
+        case SQP:
+            config->globalization->step_update = &ocp_nlp_update_variables_sqp;
+            break;
+        case SQP_RTI:
+            config->globalization->step_update = &ocp_nlp_update_variables_sqp;
+            break;
+        case DDP:
+            config->globalization->step_update = &ocp_nlp_ddp_compute_trial_iterate;
+            config->globalization->find_acceptable_iterate = &ocp_nlp_globalization_merit_backtracking_find_acceptable_iterate_for_ddp;
```

Comment:
It would make sense to assign the function `find_acceptable_iterate` in all cases here.

Reply:
I moved the find acceptable iterate into the if-statement. I would not add find acceptable iterate for all here since we would need the many additional if statements.
But it was indeed a bug to not have ddo in the if-clause